packages/main/src/Select.ts

```diff
@@ -214,6 +214,18 @@ class Select extends UI5Element implements IFormInputElement {
 	@property({ type: Boolean })
 	disabled = false;
 
+	/**
+	 * Defines the icon, displayed as graphical element within the component.
+	 * The SAP-icons font provides numerous options.
+	 *
+	 * Example:
+	 * See all the available icons within the [Icon Explorer](https://sdk.openui5.org/test-resources/sap/m/demokit/iconExplorer/webapp/index.html).
+	 * @default undefined
+	 * @public
+	 */
+	@property()
+	iconOnly?: string;
```

Comment:
The property iconOnly sounds too much like e boolean property, not a string one.

it's better to call it just icon:

```ts
@property()
icon?: string;
```

 And, when you need to use it conditions you can have getter - 
```ts
get iconOnly() { return !!this.icon;}
```